packages/node_modules/@ciscospark/react-component-utils/src/metrics.js

```diff
@@ -96,7 +95,7 @@ function getElapsedTime() {
 
 /**
  * Save time elapsed for event
- * @type {[type]}
+ * @type {type}
```

Comment:
looks like this comment block just generated completely incorrectly. probably because of the gap at line 100. you'll want `@param` and `@returns`, not `@type`